external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala

```diff
@@ -310,6 +309,20 @@ class KafkaTestUtils(withBrokerProps: Map[String, Object] = Map.empty) extends L
     offsets
   }
 
+  private def getPartitions(consumer: KafkaConsumer[String, String]): JSet[TopicPartition] = {
```

Comment:
Duplicated code. Maybe we could put this in utility class or somewhere and pass timeout as parameter?

Reply:
Originally I was thinking about to move it but not big fan of static functions. Let me check what can we do or where...

Reply:
Now test code uses `KafkaOffsetReader`.